suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -25,8 +39,6 @@ function assembleCustomHeaders(ACCESS_TOKEN) {
 
 // TODO: Consolidate some of the common scenarios across other executable tests to avoid duplicates
 // e.g., The "fetchDIDLists()" function is also included in suites/apis/dataStageOIDCFlowTest.js
-// TODO: Verify better approach to find DIDs for positive and negative tests based
-// on the "/index?acl=<acl>" API call
 
 async function fetchDIDLists(I) {
```

Comment:
you wouldn't need this function anymore since you are creating your own records

Reply:
I would need that, as I am not planning to create records for positive scenario